README.md

```diff
@@ -1,2 +1,69 @@
-# windup-operator
-Windup Operator for Openshift-Kubernetes
+# windup-operator project
+
+This project creates an Operator to deploy Windup/MTA on an Openshift cluster ( Kubernetes one in future releases)
+
+It consists on a series of yaml files to deploy the operator (CRD, role, rolebinding, serviceaccount) and the native artifact compiled using Quarkus and GraalVM
+
+At this moment the operator reacts to creation of the Windup Custom Resource and will deploy all needed objects ( deployments, ingress, services, persistent volumes)
+
+## Building and pushing the Java code manually in native mode
+1. The following command will use the configuration in the `application.properties` file :  
+```
+quarkus.kubernetes.deployment-target=kubernetes, openshift
+quarkus.native.resources.includes=k8s/def/*.yaml
+quarkus.container-image.registry=quay.io
+quarkus.container-image.group=windupeng
+quarkus.container-image.name=windup-operator-native
+quarkus.container-image.tag=latest
+quarkus.kubernetes.service-type=load-balancer
+
+```
```

Comment:
This block should be indented to avoid the list numbering to restart after this block
![Screenshot from 2020-11-17 17-57-50](https://user-images.githubusercontent.com/7288588/99421582-d6792580-28fe-11eb-81da-d9bbda022879.png)

Reply:
:+1: